components/AdminArtPieceForm/ArtPieceForm.js

```diff
@@ -0,0 +1,197 @@
+import { styled } from "styled-components";
```

Comment:
Even though it doesn't scream at you, it's actually a default export.
```suggestion
import styled from "styled-components";
```